src/state.rs

```diff
@@ -132,7 +132,17 @@ impl State {
                                 // Only perform actions if the record is saved in ledger
                                 match state.db.mark_paid(id.clone()).await {
                                     Ok(order) => {
-                                        // TODO: callback here
+                                        if let Some(callback) = order.callback.clone() {
+                                            tokio::spawn(async move {
+                                                tracing::info!("Sending callback to: {}", callback);
+                                                let client = reqwest::Client::new();
```

Comment:
is this like 3rd different entity called just "client" here?

Reply:
addressed in 4576c71